db/mysql/schema.sql

```diff
@@ -286,6 +300,16 @@ CREATE TABLE `sch_usr_notification_email`
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 CREATE INDEX `ix_SCH_USR_NOTIFICATION_EMAIL_email_id` ON `sch_usr_notification_email` (`email_id`);
 
+
+CREATE TABLE `organization`
```

Comment:
Ok, but than there should be a possibility to specify organization via GUI (in manage users section). Otherwise, the concept of organisations cannot be used anyhow in non-SSO settings. 
See the issue: https://github.com/UnifiedViews/Core/issues/341